backend/src/ee/services/dynamic-secret/dynamic-secret-service.ts

```diff
@@ -521,9 +559,10 @@ export const dynamicSecretServiceFactory = ({
     deleteByName,
     getDetails,
     listDynamicSecretsByEnv,
-    listDynamicSecretsByFolderIds,
+    listDynamicSecretsByEnvs,
```

Comment:
![Screenshot 2024-10-25 at 2 40 42 PM](https://github.com/user-attachments/assets/166adff0-9846-4ef6-a651-207de75c0ec0)

This is starting to get real messy for secret,folder and dynamic secret services. I am seeing a lot of these kinda services. We may need to soon refactor this

Reply:
I agree; we can collapse counts into the queries themselves and also probably remove individual env queries (just pass single env in array to multi)